docs/contributors/updating_kubernetes.md

```diff
@@ -44,7 +44,7 @@ Make sure to also fetch tags, as Godep relies on these.
 
  ```shell
  git checkout $DESIREDTAG
- ./hack/godeps/godep-restore.sh
+ ./hack/godep-restore.sh
```

Comment:
It looks like `./hack/godeps/godep-restore.sh` is actually the right path? `./hack/godep-restore.sh` is the path in kubernetes/kubernetes

We have a script at `./hack/godeps/godep-update-k8s.sh` that this doc should probably point to instead.

The right usage for that is `KUBE_VERSION=v1.8.1 ./hack/godeps/godep-update-k8s.sh`

Reply:
My reading of those instructions is that they refer to restoring Kubernetes' dependencies (`hack/godep-restore.sh`) and not Minikube's. Is that incorrect?